src/include/sof/drivers/ipc.h

```diff
@@ -124,6 +124,14 @@ struct ipc {
 #define ipc_get_drvdata(ipc) \
 	((ipc)->private)
 
+static inline uint64_t ipc_task_deadline(void *data)
+{
+	/* TODO: Currently it's a workaround to execute IPC tasks ASAP.
+	 * In the future IPCs should be considered a budget tasks and
+	 * deadline should be calculated based on that assumption.
```

Comment:
@tlauda "should be considered a budget tasks" does not make sense, can you elaborate a bit more?

Reply:
Explained it in more details.

Reply:
@tlauda The "considered a budget tasks" still bugs a bit. Are IPCs budget tasks, or should IPCs be allocated a cycle budget? I'm guessing you mean the latter, but the text could be clearer.